lib/index.d.ts

```diff
@@ -361,7 +361,14 @@ declare namespace Realm.Sync {
         expiresAt?: Date;
     }
 
-    type ErrorCallback = (message?: string, isFatal?: boolean, category?: string, code?: number) => void;
+    class SyncError {
+        message: string; 
+        isFatal: boolean 
+        category?: string 
+        code: number;
+    }
+    
+    type ErrorCallback = (sender, error: SyncError) => void;
```

Comment:
`sender` should be `sender: Session`, I think.

Reply:
thanks, if we are sure this is always the case I can do it. I was thinking to do it `sender: any` and then i missed to do it. 
Why not call it `session: Session` then